record_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sdk
+
+import (
+	"testing"
+
+	"github.com/matryer/is"
+)
+
+func TestRecord_Bytes(t *testing.T) {
+	is := is.New(t)
+
+	r := Record{
+		Position:  Position("foo"),
+		Operation: OperationCreate,
+		Metadata: map[string]string{
+			MetadataConduitPluginName: "example",
+		},
+		Key: RawData("bar"),
+		Payload: Change{
+			Before: nil,
+			After: StructuredData{
+				"foo": "bar",
+				"baz": "qux",
+			},
+		},
+	}
+
+	want := `{"position":"Zm9v","operation":"create","metadata":{"conduit.plugin.name":"example","opencdc.version":"v1"},"key":"YmFy","payload":{"before":null,"after":{"baz":"qux","foo":"bar"}}}`
+
+	got := string(r.Bytes())
+	is.Equal(got, want)
```

Comment:
Nitpick: we may want to verify the OpenCDC version is not left behind in the metadata.

Reply:
Done!